### DISCUSSION TITLE
Access

### DISCUSSION THREAD

**Discussion by aarushi102** [community/community#66088] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am working on github for some repositories as a part of my organizaion. I have been encountering a problem that all the members of the organization can see all the repositories and can even pull code. But I want only specific members that is the people who are the part of that repository or the ones who have been added to that repository can see it. How do I do that what can be the steps so that the repositories are not affected and people can only see those repositories which they are a part of and not all the repositories. Please help

**Comment by arafajs** [comment, marked as answer]
Unfortunately, there is no direct way to do this, but you can try this:

- Make the repositories private. By default, repositories in an organization are visible to all members of the organization. Making a repo private will restrict access to only people you explicitly give access to.
- Manage team and collaborator access. Create teams within your organization and give those teams access to specific repositories they need. You can also add individual collaborators to repositories and give them read, write, or admin access.
- Use repository permission settings. In the settings for each repository, go to the "Collaborators & teams" section. Here you can specify which users and teams have access to the repo. Remove general organization teams if needed.
- Remove outside collaborators. Check the collaborators list for each repository and remove any that should not have access.
- Leverage organization teams properly. Put users in the appropriate organization teams based on which repos they should see. Adjust team access as needed.

**Reply by aarushi102** [reply]
The repositories are already private and we have made a specific team for  specific projects but still every person having the link for one repository they can still see other repository as well

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @aarushi102! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
